test/passing/doc_comments-after.ml.ref

```diff
@@ -256,12 +256,12 @@ exception A of int
 
 module A = struct
   module B = struct
-    (** It does not try to saturate (2) A = B + C /\ B = D + E => A = C + D
-        \+ E Nor combine more than 2 equations (3) A = B + C /\ B = D + E /\
-        F = C + D + E => A = F
+    (** It does not try to saturate (2) A = B + C /\ B = D + E => A = C + D +
```

Comment:
This test is broken, the tested feature was to escape the `+` at the beginning of a line.
It fails.

Reply:
I didn't manage to fix that.
An other break hint is inserted before the `cbreak` you added, it seems, so it's never breaking.
I'd suggest to revert this changes.

Reply:
I will have a look

Reply:
Somehow the output can get closer to the margin, and since there is a break between each word we just break after `+` instead of before, I just made the part before the original break longer to output the escape character.